kv/local_test_cluster.go

```diff
@@ -146,5 +122,8 @@ func (ltc *LocalTestCluster) Start(t util.Tester) {
 
 // Stop stops the cluster.
 func (ltc *LocalTestCluster) Stop() {
+	if r := recover(); r != nil {
```

Comment:
I've tried it after seeing it in your PRs, and had mixed results with it. Sometimes it helps, by showing the original panic instead of a timeout, and sometimes it hurts, by losing the original panic's stack trace. I don't think it's worth adding this to all our stop methods; when I'm debugging something that might be affected by this I just comment out the `defer Stop()` line.